Doc/deprecations/pending-removal-in-future.rst

```diff
@@ -153,4 +153,4 @@ although there is currently no date scheduled for their removal.
   ``elem is not None`` tests instead.
 
 * :meth:`zipimport.zipimporter.load_module` is deprecated:
-  use :meth:`~zipimport.zipimporter.exec_module` instead.
```

Comment:
Stray newline change! We're not touching the file otherwise, so could this be undone? Thanks

Reply:
My bad! It's fixed.